src/docs/howto-content/article-blog/index.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Create a library of articles
+slug: article-library
+updated: 2022-12-22
+category: how-to-content
+ingress:
+  With the Flex Pages feature, you can build static pages on your
+  marketplace directly from Flex Console. This how-to guide shows how to
+  create a blog-style library of interlinked articles on your
+  marketplace.
+published: true
+---
+
+The Flex Pages feature allows you to create different types of static
+content. Using the different content elements – article, feature, column
+– you can build a wide range of different types of pages.
+
+One use case for creating static content is to have a library of
```

Comment:
max 100 Pages, so not a blog in the sense that you can keep creating posts e.g. every day, but more of a library

Reply:
A note of this exists in the article blog how-to